### DISCUSSION TITLE
Uninstall GDM Setting Previously Installed by Source

### DISCUSSION THREAD

**Discussion by ghost** [realmazharhussain/gdm-settings#205] [Q&A] [answered]
I installed “GDM Settings” from source, but unfortunately It's not working, so I want to uninstall it, but I'm completely unfamiliar with meson and ninja build tools, **How can I Uninstall GDM Setting previously installed by source?**

**Comment by realmazharhussain** [comment, marked as answer]
The instructions are privided [here](https://github.com/gdm-settings/gdm-settings/wiki/Uninstall#manually-installed).

**Reply by realmazharhussain** [reply]
Feel free to ask further questions if this doesn't work for you.

**Reply by ghost** [reply]
Realy sorry I missed that part of docs... Thanks

**Reply by realmazharhussain** [reply]
No problem at all.
